### DISCUSSION TITLE
Featured in JayzTwoCents' video

### DISCUSSION THREAD

**Discussion by Paft** [Rem0o/FanControl.Releases#585] [General]
https://www.youtube.com/watch?v=uDPKVKBMQU8

**Comment by wakkawakka22** [comment]
Congrats!

**Comment by MemberLogic** [comment]
Haven't even downloaded yet but came here to donate from JayzTwoCents' video. Mixed curve fan control from cpu/gpu is exactly what I need for my sff build. Plan on downloading this weekend!

**Comment by Technous285** [comment]
Coming over here from Jay's video, dropped a fiver through the donation as I see plenty of present and future value in the project having been looking for a replacement for SpeedFan for my Win10/11 machine for ages!
Currently got the prog doing the first-time setup and calibration tests as I'm typing this.

**Comment by darksplice** [comment]
How to download it?

**Reply by Technous285** [reply]
Download the "source code" zip file, then uncompress it with like 7Zip (or WinRAR or WinZip), then uncompress the "fancontrol.zip" file within that uncompressed folder to decompress the actual program.

**Reply by darksplice** [reply]
Thanks, I did not know how, Cheers!

**Comment by MrBuckRogers** [comment]
I came from Jayz channel as well! Keep up the good work! Here's $20 for you to keep going.

**Comment by MoridinTX** [comment]
Ditto, came from Jayz, awesome work!

**Comment by Rem0o** [comment]
> https://www.youtube.com/watch?v=uDPKVKBMQU8

My inbox simply exploded since a few hours ago, hard to miss out on 😅

Thanks everyone for the love and support!

**Reply by GodlyOverlord** [reply]
Fantastic software, i'd never have known about it if it wasn't for the video but its literally everything i ever wanted to just control my damn fans lol. Keep it up!

**Reply by AtA3301** [reply]
> > https://www.youtube.com/watch?v=uDPKVKBMQU8
> 
> My inbox simply exploded since a few hours ago, hard to miss out on 😅
> 
> Thanks everyone for the love and support!

Within hours we got a new plugin too - amazing. Jay inspired me to sponsor you - been using FanControl for a while now, so a bit overdue ;-)

**Comment by jayfroelich** [comment]
This is exactly what I have been looking for. I just finished my PC build... It has 3 case fans, 6 fans mounted to two radiators, 3 GPU fans, and a water pump. I can now control each of the fans or pump the way the need to be ran.

Thank you
